TSC-Charter.md

```diff
@@ -74,6 +74,7 @@ including:
 * Development process and any coding standards.
 * Mediating technical conflicts between Collaborators or Foundation
   projects.
+* Communication about the above that would be considered official, and the development process of said communication.
```

Comment:
Or actually, that might be a bit too meta, there is:

1. Governance of technical things A, B, C, D
2. Technical things A, B, C, D

Trying to find a wording that covers both 1 and 2, because the above points are mostly about 1..

```suggestion
* Official communication about the above and what they govern, and the development process of said communication.
```